doc/examples/lm/ray_train.sh

```diff
@@ -4,7 +4,6 @@ TOTAL_UPDATES=125000       # Total number of training steps
 WARMUP_UPDATES=10000       # Warmup the learning rate over this many updates
 PEAK_LR=0.0005             # Peak learning rate, adjust as needed
 TOKENS_PER_SAMPLE=512      # Max sequence length
-MAX_POSITIONS=512          # Num. positional embeddings (usually same as above)
```

Comment:
Is this unused? Just making sure the change is intentional (same as below).

Reply:
Yup this was intentional, they do appear to be unused.

Reply:
Also I merged in master but CI seems clogged, here's the build on my fork: https://github.com/mehrdadn/ray/runs/906384304?check_suite_focus=true